.github/workflows/label-ready-approved.yml

```diff
@@ -0,0 +1,13 @@
+on: pull_request_review
+name: Label Ready to Test
+jobs:
+  label:
+    name: When Approved
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label when approved
+        uses: abinoda/label-when-approved-action@v1.0.7
+        env:
+          APPROVALS: "2"
```

Comment:
Looking at the docs, I can't tell if these are CODEOWNER approvals or any general approvals (I suspect second, we'll see after merge). Either way, probably still right to set it at 2.

Reply:
From reading the code [1] it's any approvals, but I think it's fine for this context anyhow since it will still save running all the tests when the PR is fresh and still under review.

[1] https://github.com/abinoda/label-when-approved-action/blob/master/entrypoint.sh#L49